environments/backup-production/terraform.yml

```diff
@@ -108,7 +112,7 @@ servers:
       volume_size: 20
       encrypted: yes
     group: "couchdb2"
-    os: bionic
+    os: bionic # should we change this?
```

Comment:
Yeah, let's

Reply:
Let's change every occurrence of bionic to jammy, regardless of what's on production

Reply:
https://github.com/dimagi/commcare-cloud/pull/6157/commits/746adc606be43c1e233c665d2dda4ae41e68e7af